src/ui_fsmenu/addons/manager.cc

```diff
@@ -610,7 +610,7 @@ AddOnsCtrl::AddOnsCtrl(MainMenu& fsmm, UI::UniqueWindow::Registry& reg)
 			}
 		}
 		assert(!upgrades.empty());
-		if (nr_full_updates > 0 && (!all_verified || ((SDL_GetModState() & KMOD_CTRL) == 0))) {
+		if (nr_full_updates > 0 && (!all_verified || !matches_keymod(SDL_GetModState(), KMOD_CTRL))) {
```

Comment:
That's strange, I deleted this one, when I realised that it checks for lack of CTRL, not presence.

Please note the slightly changed semantic: `& KMOD_…` checks only whether the given modifier is pressed, regardless of the others, while `matches_keymod()` checks for exact matches (considering left/right modifier equivalence).

So if it's OK here, there are a few others like this, which could also use `!matches_keymod(KMOD_CTRL)`. But also a few where it accepts either CTRL or SHIFT, which I think can be left alone.